app/static/src/js/headscripts/gtm.js

```diff
@@ -23,6 +23,45 @@ function add_GTM() {
     GTM_Loaded = true;
 }
 
+function push_to_datalayer(params) {
+    window.dataLayer = window.dataLayer || [];
+    window.dataLayer.push({ ...params });
+}
+
+function diagnosed(){
+    const path = window.location.pathname;
+    const categoryData = {
+        event: 'diagnosed',
+        category_code: window.sessionData?.category_code || '',
+        category_name: window.sessionData?.category_name || '',
+        category_traversal: window.sessionData?.category_traversal || [],
+    };
+
+    let diagnosis_result = null;
+
+    // Covers the in scope legal aid available page, the fast tracked contact
+    if (path.endsWith('/legal-aid-available') || path.includes('fast-tracked')) {
+        diagnosis_result = "INSCOPE";
+    }
+    // Covers the refer page and FALA search
+    else if (path.endsWith('/cannot-find-your-problem') || path.includes('/find-a-legal-adviser')) {
+        diagnosis_result = "OUTOFSCOPE";
+    }
+
+    if (diagnosis_result !== null) {
+        push_to_datalayer({
+            ...categoryData,
+            diagnosis_result
+        });
+    }
+}
+
+// Diagnosed Events
+document.addEventListener('DOMContentLoaded', function () {
+    diagnosed();
```

Comment:
Something here is causing the event to register on GTM on every page. Not sure why yet.